pages/installing_new_languages.md

```diff
@@ -26,46 +26,47 @@ These are the currently supported languages
 *   Albanian
 *   Arabic (Libya)
 *   Armenian
-*   Belarusian
+*   Belorussian
```

Comment:
This is not correct, it is called `Belarusian`

Reply:
[Belarusian is also spelled as Belorussian](https://en.wikipedia.org/wiki/Belarusian_language#Names). :smile: